tests/foreman/ui/test_http_proxy.py

```diff
@@ -251,29 +249,30 @@ def test_set_default_http_proxy(module_org, module_location, setting_update, tar
 def test_check_http_proxy_value_repository_details(
     function_org, function_location, function_product, setting_update, target_sat
 ):
-    """Deleted Global Http Proxy is reflected in repository details page".
+    """Global Http Proxy is reflected in repository details page".
 
     :id: 3f64255a-ef6c-4acb-b99b-e5579133b564
 
     :steps:
         1. Create Http Proxy (Go to Infrastructure > Http Proxies > New Http Proxy)
-        2. GoTo to Administer > Settings > content tab
+        2. GoTo to Administer > Settings > Content tab
         3. Update the "Default HTTP Proxy" with created above.
-        4. Create repository with Global Default Http Proxy.
-        5. Delete the Http Proxy
+        4. Create repository, check the Global Default Http Proxy is used.
+        5. Delete the Http Proxy.
+        6. Check it no longer appears on the Settings and repository page.
 
     :BZ: 1820193
 
     :parametrized: yes
 
     :expectedresults:
-        1. After deletion of  "Default Http Proxy" its field on settings page should be
-            set to no global defult
-        2. "HTTP Proxy" field  in repository details page should be set to Global Default (None).
+        1. Repository is automatically created with relevant Global Default Http Proxy.
+        2. After Http Proxy deletion
+           - its field on Settings page should be set to Empty.
```

Comment:
Should it be empty or None?

Reply:
It shows "Empty" in Settings > Content when not set.